roles/cifmw_cephadm/tasks/pre.yml

```diff
@@ -158,15 +158,3 @@
     mode: "0644"
     config_overrides: "{{ cifmw_cephadm_conf_overrides }}"
     config_type: ini
-
```

Comment:
The stat'ing of the certificate paths is now handled by `roles/cifmw_external_dns/tasks/requirements.yml`